sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/TableServiceDataTypes.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.tables.implementation;
+
+/**
+ * Table service data types.
+ */
+public abstract class TableServiceDataTypes {
```

Comment:
I don't see this type being used anywhere other than in `EntityDataModelType`, does this class need to exist to hold these constants or can they be constants in `EntityDataModelType` instead?